test/libyul/yulOptimizerTests/redundantStoreEliminator/unrelated_relative.yul

```diff
@@ -0,0 +1,21 @@
+{
+    let c := calldataload(0)
+    mstore(c, 4)
+    mstore(add(c, 0x20), 8)
+    sstore(0, mload(c))
+    mstore(c, 9)
+    mstore(add(c, 0x20), 20)
+}
+// ----
+// step: redundantStoreEliminator
+//
+// {
+//     let c := calldataload(0)
+//     mstore(c, 4)
+//     let _3 := 8
+//     mstore(add(c, 0x20), _3)
+//     sstore(0, mload(c))
```

Comment:
Will have to investigate why this is not replaced by `4` and the mstore not removed.

Reply:
Did you find out?

Reply:
I think this is because the component only works with CSE run before it. If this is moved to `fullSuite` then it should work. I will re-check.

Reply:
Now it gets even weirder...

Reply:
ah no, now it's fine :)